index.html

```diff
@@ -135,38 +137,42 @@
                     It can ingest data from offline data sources (such as Apache Hadoop and flat files) as well as online sources (such as Apache Kafka).
                     Pinot is designed to scale horizontally.</p>
                 <p class="lead">
-                    <a class="btn btn-default btn-animated btn-outline" >Download</a>
-                    <a class="btn btn-default btn-animated btn-outline" href="https://pinot.readthedocs.io/" target="_blank">Try Pinot</a>
+                    <a class="btn btn-default btn-animated btn-outline" href="https://pinot.readthedocs.io/en/latest/intro.html" target="_blank">Learn More</a>
+                    <a class="btn btn-default btn-animated btn-outline" href="#">Download</a>
                 </p>
+                <h3 style="margin-top: 0px; "><a href="https://cwiki.apache.org/confluence/display/PINOT/Announcements" target="_blank"><strong>See Latest News</strong></a></h3>
             </div>
             <!--===============================================
                Pinot Primary Content <end>
             =================================================-->
-            <!--===============================================
-               Pinot News Content <start>
-            =================================================-->
-            <div class="col-lg-4 col-md-5 col-sm-6">
+
+        </div>
+        <!--===============================================
+          ROW 1 <end>
+        =================================================-->
+        <!--===============================================
+          ROW 2 <start>
+        =================================================-->
+        <div class="row" style="background-color: #e8e8e8; padding-bottom: 10px; margin-top: 15px;">
+            <div class="col-sm-12">
+                <h2 class="sectiontile" style="text-align: center">Powered By Pinot</h2>
                 <div class="row">
-                    <div class="col-sm-12 panel panel-default latest-news-panel">
-                        <p class="latest-news-header">Latest News</p>
-                        <div id="news-archive-link">
-                            <p>Coming soon!</p>
-                        </div>
-                    </div>
+                    <a href="http://www.linkedin.com/" style="margin-left: 100px;" target="_blank"><img style="width: 125px" src="assets/images/Logo-2C-128px-R.png"></a>
+                    <a href="http://www.slack.com/" style="margin-left: 150px;" target="_blank"><img style="width: 145px" src="assets/images/Slack_RGB.png"></a>
+                    <a href="http://www.uber.com/" style="margin-left: 150px;" target="_blank"><img style="width: 125px" src="assets/images/Uber_Logo_Black.png"></a>
+                    <a href="https://www.microsoft.com/" style="margin-left: 150px;" target="_blank"><img style="width: 50px" src="assets/images/Teams_Logo_Only_Blurple.svg"></a>
```

Comment:
I'm not sure about these logos on the front page. might cause confusion on the branding. perhaps move this to a subpage? also this could open up in the future for other companies to ask to be added

Reply:
I thought about it too but since there are only a few companies right now, having it's own page looks pretty empty. We can definitely move this to a sub page once we have more companies participating.

Reply:
do you have a screen shot on what the page looks like?

Reply:
Added it to the description

Reply:
ok